### PR TITLE
Add Windows Development Guide

### PR DESCRIPTION
**Pull request by charlieJ107**
Migrate Windows Developer Guide from orca-lite.

### CODE REVIEW

File: python/nano/Windows_Development_Guide.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Windows development guide
```
Comment (yangw1234) [inline python/nano/Windows_Development_Guide.md:1]: Let's rename it to "Windows User Guide (WSL2)"?

File: python/nano/Windows_Development_Guide.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+```bash
+conda create -n bigdl-nano
+conda activate bigdl-nano
+```
+
+
+## Step 4: Install BigDL Nano from Pypi
+
+You can install BigDL nano from Pypi with `pip`.
```
Comment (yangw1234) [inline python/nano/Windows_Development_Guide.md:40]: Let's use different commands for pytorch and tensorflow

File: python/nano/Windows_Development_Guide.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Windows User Guide (WSL2)
```
Comment (yangw1234) [inline python/nano/Windows_Development_Guide.md:1]: Also file name

File: python/nano/Windows_Development_Guide.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+## Step 1: Install WSL2
+
+BigDL-Nano supports windows systems through *Windows Subsystem for Linux 2* (WSL2).
```
Comment (yangw1234) [inline python/nano/Windows_Development_Guide.md:5]: For Windows, we recommend using *Windows Subsystem for Linux 2* (WSL2) to run BigDL-Nano.

File: python/nano/Windows_Development_Guide.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+# Windows User Guide (WSL2)
+
+## Step 1: Install WSL2
+
+BigDL-Nano supports windows systems through *Windows Subsystem for Linux 2* (WSL2). 
+
+To install WSL2, you can now install everything you need by entering this command in an administrator PowerShell or Windows Command Prompt and then restarting your machine.
```
Comment (yangw1234) [inline python/nano/Windows_Development_Guide.md:7]: Specify windows version requirements.

File: python/nano/Windows_Development_Guide.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+conda activate bigdl-nano
+```
+
+
+## Step 4: Install BigDL Nano from Pypi
+
+You can install BigDL nano from Pypi with `pip`.
+
+```
+pip install bigdl-nano
```
Comment (yangw1234) [inline python/nano/Windows_Development_Guide.md:43]: let's remove this.

**Comment by charlieJ107** [comment]
Updated in https://github.com/intel-analytics/BigDL/pull/3842/commits/9b88df79f6e03781df9ffaf5c8fb9322e15f05e6

* Removed the `pip install bigdl-nano` command
* Specificed the Windows version requirements
* Rename the file

**Comment by yangw1234** [comment]
LGTM

**Pull request merged**